crates/router/tests/connectors/payme.rs

```diff
@@ -411,7 +494,7 @@ async fn should_fail_for_refund_amount_higher_than_payment_amount() {
         .unwrap();
     assert_eq!(
         response.response.unwrap_err().message,
-        "Refund amount (₹1.50) is greater than charge amount (₹1.00)",
+        "internal_server_error",
```

Comment:
why internal_server_error in all the tests, is this a closed connector?

Reply:
connector always throws 500, with error message in Hebrew language. Since it is 500 core is treating it as internal_server_error. The default behavior for error scenario is not known since we don't have open docs.

`{
    "status_code": 1,
    "status_error_details": "נא לבדוק תקינות קישורים",
    "status_additional_info": "http://localhost",
    "status_error_code": 21,
    "session": "sz4rn0to"
}`